CONTRIBUTING.md

```diff
@@ -37,7 +37,8 @@ Contact us at [quire@getty.edu](mailto:quire@getty.edu)
 
 There are two main code repositories for Quire hosted on GitHub.
 
-- [**quire**](https://github.com/thegetty/quire): The command-line interface for Quire, default starter content, and default theme.
+- [**quire**](https://github.com/thegetty/quire): Quire’s primary repository, with the command-line interface and 11ty template packages
+- [**quire-starter-default**] The default starter content for a Quire project 

```

Comment:
Make this a link too? https://github.com/thegetty/quire-starter-default

And don't forget the colon after the repo name/link. It's missing currently